tasks/test-changes.js

```diff
@@ -38,7 +38,8 @@ const getChangedPackages = () => {
                 // There are no benchmarks available in this directory.
                 item.location.search('projects') === -1 &&
                 // The icons-* tests are particular and long, exclude in CI.
-                !name.startsWith('icons-')
+                !name.startsWith('icons-') &&
+                !name.startsWith('color-field')
```

Comment:
Doesn't let me comment on line 51, but you could return `['overlay']` and hopefully only need to run a single test each time to see what the output would be.